client/my-sites/themes/theme-options.js

```diff
@@ -74,6 +77,34 @@ const activateOnJetpack = {
 	)
 };
 
+const deleteConfirmation = ( themeId, siteId ) => {
+	return ( dispatch, getState ) => {
+		const { name: themeName } = getTheme( getState(), siteId, themeId );
+		const siteSlug = getSiteSlug( getState(), siteId );
+		accept(
+			i18n.translate(
+				'Are you sure you want to delete %(themeName)s from %(siteSlug)s?',
+				{ args: { themeName, siteSlug }, context: 'Themes: theme delete confirmation dialog' }
+			),
+			( accepted ) => {
+				accepted && dispatch( deleteThemeAction( themeId, siteId ) );
+			},
+			i18n.translate(
+				'Delete %(themeName)s',
+				{ args: { themeName }, context: 'Themes: theme delete dialog confirm button' }
+			),
+			i18n.translate( 'Back', { context: 'Theme: theme delete dialog back button' } )
```

Comment:
Alternate string suggestion: 
* `translate( 'Back', { context: 'go back (like the back button in a browser)'} )`&mdash; translations: **35**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/theme-delete-ui)